docs/guides/08-introduction-to-sprites.md.WIP

```diff
@@ -0,0 +1,65 @@
+## Introduction to Sprites
+
+CARTO VL has support for sprites, replacing the typical points by images. Those images can be builtin or external and they can also be colored and placed by CARTO VL expressions.
+
+### Built-in sprites
+
+We provide a set of built-in images ready to be used without additional HTTP requests. You can use the `symbol` property with the name of a built-in sprite to see it.
+```js
+symbol: cross
+```
+
+http://127.0.0.1:8080/examples/editor/index.html#eyJhIjoic3BlbmRfZGF0YSIsImIiOiIiLCJjIjoiY2FydG92bCIsImQiOiJodHRwczovL3t1c2VyfS5jYXJ0by5jb20iLCJlIjoid2lkdGg6IDcwXG5jb2xvcjogcmdiKDIwNCwwLDApXG5zeW1ib2w6IGNyb3NzIiwiZiI6eyJsbmciOjIuMTY3NjA2NDU4ODExMTc4NSwibGF0Ijo0MS40MDgwMTMzNzY3NDIyNX0sImciOjE3LjA0MjkyMzM2MTQxOTIxLCJoIjoiRGFya01hdHRlciJ9
+
+### External sprites
+
+Images can be loaded by providing URL links to them. Keep in mind that the server must respond with the appropriate CORS headers.
+
+You can use the `sprite` expression to create a new sprite with an external image:
+```js
+symbol: sprite('https://libs.cartocdn.com/carto-vl/assets/marker.svg')
+```
+
+http://127.0.0.1:8080/examples/styling/sprite.html
+
+### Matching categories with sprites
+
+You can use multiple sprites by assigning different sprites to different points, matching by a categorical expression.
+
```

Comment:
* matching by a categorial expression -->
    * that matches a categorical expression (?)
    * matching a categorial expression (?)